terraform/monitoring/03_services.tf

```diff
@@ -86,3 +86,66 @@ resource "google_monitoring_custom_service" "custom_service" {
   service_id = "${var.custom_services[count.index].service_id}-srv"
   display_name = var.custom_services[count.index].service_name
 }
+
+# Define services that will use the Istio service that is automatically detected
```

Comment:
I find this confusing.  Why would you have to define a service if it's already been detected?  I didn't think you had to do that with service monitoring.

Reply:
Yes I will change the wording here - the user isn't creating any services they are just specifying which of the 10 services will use the Istio services vs which will use the custom service. I'll change the word 'Define' to 'Specify'